src/plone/restapi/serializer/relationfield.py

```diff
@@ -33,4 +33,18 @@ class RelationChoiceFieldSerializer(DefaultFieldSerializer):
 @adapter(IRelationList, IDexterityContent, Interface)
 @implementer(IFieldSerializer)
 class RelationListFieldSerializer(DefaultFieldSerializer):
-    pass
+    def get_value(self, default=[]):
+        """Return field value reduced to list of non-broken Relationvalues.
+
+        Args:
+            default (list, optional): Default field value. Defaults to empty list.
+
+        Returns:
+            list: List of RelationValues
+        """
+        value = getattr(
+            self.field.interface(self.context), self.field.__name__, default
+        )
```

Comment:
I would use `super` to call the method inherited from DefaultFieldSerializer here, instead of copying its code: 

```suggestion
        value = super().get_value(default)
```